crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -19,6 +19,7 @@
 require "json"
 
 class BarclampController < ApplicationController
+  skip_before_filter :enforce_installer
```

Comment:
So we have to put this in all controllers? Isn't the bit from `application_controller.rb` enough?